refinery/tool_manager/tests.py

```diff
@@ -1653,7 +1653,7 @@ def test_tool_launch_has_resonable_default_display_name(self):
         self.create_tool(tool_type=ToolDefinition.VISUALIZATION)
         self.assertEqual(self.tool.display_name, "{} {} {}".format(
```

Comment:
I would prefer to have static string here, if possible, rather than recapitulating the logic from the code.